exercises/php/cards/PlayingCard.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+namespace McrDigital\PhpFundamentals1\Cards;
+
+use Exception;
+
+class PlayingCard
+{
+    private string $suit;
+    private int $faceValue;
+
+    public function __construct(string $suit, int $faceValue)
+    {
+        $this->suit = $suit;
+        $this->faceValue = $faceValue;
+    }
```

Comment:
Is it possible for the `$suit` member to be of type Suit rather than type String? At the moment, I think the Suit class is just used as a place to store the array of suits. I’m thinking it should have a `$name` member (or perhaps `$suitname`) of type `string`, and then it could be used to provide a type to the member in `PlayingCard`. That’s the way we’ve done it in Java, but I don’t know if there’s a PHP reason not to do this 😃